release-prepare-perform.sh

```diff
@@ -11,6 +11,8 @@ REMOTE_REPOSITORY="${bamboo_planRepository_repositoryUrl}"
 
 RELEASE_PLUGIN="org.apache.maven.plugins:maven-release-plugin:2.5.1"
 
+SEMVER_REGEX="^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(-((0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(\.(0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*)){0,1}(\+([0-9a-zA-Z-]+(\.[0-9a-zA-Z-]+)*)){0,1}$"
+
```

Comment:
Oh wow, that's a long regex! 
Can we add a couple of comments with the things that should pass?

Reply:
Done. This is just the regex from [here](https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string) with the non-capturing groups removed (since POSIX extended regexps don't have that concept).

Reply:
@ibacher  , there seems to be an error in the regex , all proper versions are failing to be released